src/unity/lib/unity_sframe.cpp

```diff
@@ -2016,27 +2198,39 @@ void unity_sframe::explore(const std::string& path_to_client, const std::string&
       }
 
       // parse the message as json
-      flex_int start = -1,
-               end = -1;
+      flex_int start = -1, end = -1, index = -1;
+      std::string column_name;
+
+      enum MethodType {GetRows = 0, GetModal = 1};
```

Comment:
Let's be consistent with Accordion everywhere (vs. Modal).

Reply:
sounds good